downstream/modules/platform/con-controller-instance-groups.adoc

```diff
@@ -0,0 +1,30 @@
+[id="controller-instance-groups"]
+
+= Instance groups
+
+Instances can be grouped into one or more instance groups. 
+Instance groups can be assigned to one or more of the following listed resources:
+
+* Organizations
+* Inventories
+* Job templates
+
+When a job associated with one of the resources executes, it is assigned to the instance group associated with the resource. 
+During the execution process, instance groups associated with job templates are checked before those associated with inventories. 
+Instance groups associated with inventories are checked before those associated with organizations.
+Therefore, instance group assignments for the three resources form the hierarchy: 
+
+Job Template > Inventory > Organization
+
+Consider the following when working with instance groups:
+
+* You can define other groups and group instances in those groups. 
+These groups must be prefixed with `instance_group_`. 
+Instances are required to be in the `automationcontroller` or `execution_nodes` group alongside other `instance_group_ groups`. 
```

Comment:
Should this be ".. other `instance_group_` groups." instead of ".. other `instance_group_ groups`. ?